pyqtgraph/graphicsItems/ViewBox/tests/test_ViewBox.py

```diff
@@ -58,28 +65,27 @@ def test_ViewBox():
     assertMapping(vb, view1, size1)
     
     # test tall resize
-    win.resize(400, 800)
-    app.processEvents()
-    w = vb.geometry().width()
-    h = vb.geometry().height()
-    view1 = QRectF(0, -5, 10, 20)
-    size1 = QRectF(0, h, w, -h)
-    assertMapping(vb, view1, size1)
+    # win.resize(400, 800)
```

Comment:
Ah.. yes, there are a few lines above that are required. Should start like this:

```
def test_ViewBox():
    qtest = pg.Qt.QtTest.QTest
    global app, win, vb
    QRectF = pg.QtCore.QRectF
    
    win = pg.GraphicsWindow()
    win.ci.layout.setContentsMargins(0,0,0,0)
    win.resize(200, 200)
    win.show()
    vb = win.addViewBox()
    
    # set range before viewbox is shown
    vb.setRange(xRange=[0, 10], yRange=[0, 10], padding=0)
    
    # required to make mapFromView work properly.
    qtest.qWaitForWindowShown(win)
    
    g = pg.GridItem()
    vb.addItem(g)
    win.resize(400, 400)
    vb.setAspectLocked()
    win.resize(800, 400)
    app.processEvents()

    # commented test code here
```